src/patternfly/components/Menu/menu.scss

```
@@ -296,10 +330,36 @@
 
 // Header
 .pf-c-menu__header {
+  --pf-c-menu__item--PaddingTop: var(--pf-c-menu__header--c-menu__item--PaddingTop);
+  --pf-c-menu__item--PaddingRight: var(--pf-c-menu__header--c-menu__item--PaddingRight);
+  --pf-c-menu__item--PaddingBottom: var(--pf-c-menu__header--c-menu__item--PaddingBottom);
+  --pf-c-menu__item--PaddingLeft: var(--pf-c-menu__header--c-menu__item--PaddingLeft);
+
   padding-top: var(--pf-c-menu__header--PaddingTop);
   padding-right: var(--pf-c-menu__header--PaddingRight);
   padding-bottom: var(--pf-c-menu__header--PaddingBottom);
   padding-left: var(--pf-c-menu__header--PaddingLeft);
+
+  > .pf-c-menu__item {
+    --pf-c-menu__item--BackgroundColor: var(--pf-c-menu__header--c-menu__item--BackgroundColor);
+
+    margin-top: calc(var(--pf-c-menu__header--PaddingTop) * -1 / 2);
+    margin-right: calc(var(--pf-c-menu__header--PaddingRight) * -1);
+    margin-bottom: calc(var(--pf-c-menu__header--PaddingBottom) * -1 / 2);
+    margin-left: calc(var(--pf-c-menu__header--PaddingLeft) * -1);
+
+    &:hover {
+      --pf-c-menu__item--BackgroundColor: var(--pf-c-menu__header--c-menu__item--hover--BackgroundColor);
+    }
+
+    &:focus {
+      --pf-c-menu__item--BackgroundColor: var(--pf-c-menu__header--c-menu__item--focus--BackgroundColor);
+    }
+  }
+
+  + .pf-c-menu__search {
+    --pf-c-menu__search--PaddingTop: var(--pf-c-menu__header--c-menu__search--PaddingTop);
```

This selector isn't inside of a nested menu? Should it be? Or should we update the var name?

```suggestion
    --pf-c-menu__search--PaddingTop: var(--pf-c-menu__header__search--PaddingTop);
```